src/EnergyPlus/Furnaces.cc

```diff
@@ -866,6 +885,8 @@ namespace Furnaces {
 		int SteamIndex; // steam coil index
 		Real64 SteamDensity; // density of steam at 100C
 		int DXCoilIndex; // Index to DX coil in HXAssited object
+		std::string IHPCoilName;//IHP cooling coil name
+		int IHPCoilIndex(0);//IHP cooling coil id
```

Comment:
better to start a new IHPCoilName and IHPCoilIndex, since IHP is a collection of coils, than a single cooling or heating coil name.